### PR TITLE
[SMALLFIX] Improve some worker metrics names and desc

### PR DESCRIPTION
**Pull request by jiacheliu3**
Updated some variable names and metric key descriptions

### CODE REVIEW

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 1]
```diff
@@ -1055,28 +1055,28 @@ public MetricKey build() {
...
+      new Builder("Worker.BlockRemoverBlocksRemovedCount")
+          .setDescription("The total number of blocks successfully removed from this worker "
               + "by asynchronous block remover.")
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
   public static final MetricKey WORKER_BLOCK_REMOVER_TRY_REMOVE_BLOCKS_SIZE =
       new Builder("Worker.BlockRemoverTryRemoveBlocksSize")
-          .setDescription("The size of blocks to be removed from this worker "
+          .setDescription("The number of blocks to be removed from this worker at this moment "
```
Comment (HelloHorizon) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:1072]: (nit) also change the metric name?

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 2]
```diff
@@ -1055,28 +1055,28 @@ public MetricKey build() {
...
               + "by asynchronous block remover.")
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
   public static final MetricKey WORKER_BLOCK_REMOVER_REMOVING_BLOCKS_SIZE =
       new Builder("Worker.BlockRemoverRemovingBlocksSize")
-          .setDescription("The size of blocks is removing from this worker "
+          .setDescription("The size of blocks is being removed from this worker at this moment "
```
Comment (HelloHorizon) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:1079]: Can you elaborate what is the meaning of `at this moment`?

**Review by HelloHorizon — CHANGES_REQUESTED** [review]
@jiacheliu3 Thanks for the improvement. Left some minor questions.

**Review by ggezer — APPROVED** [review]

File: core/common/src/main/java/alluxio/metrics/MetricKey.java (hunk 2 above)
Comment (jiacheliu3) [reply core/common/src/main/java/alluxio/metrics/MetricKey.java:1079]: @HelloHorizon This metric is a gauge which means the value is checked only when queried. When you poke the worker metrics and see a value from this metric, that means the value "at this moment".

File: core/common/src/main/java/alluxio/metrics/MetricKey.java (hunk 1 above)
Comment (jiacheliu3) [reply core/common/src/main/java/alluxio/metrics/MetricKey.java:1072]: @HelloHorizon This is essentially the queue size (of the to-be-removed) blocks at a moment. I think the metric name makes sense to me so i will leave it as-is.

**Review by HelloHorizon — APPROVED** [review]

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Comment by HelloHorizon** [comment]
alluxio-bot, cherry-pick this to branch-2.6 please

**Pull request merged**